CHANGELOG.latest.md

```diff
@@ -1,2 +1,2 @@
-- Fix purchasing of a product not initiating the purchase in iOS (https://github.com/RevenueCat/purchases-hybrid-common/pull/146/)
-- Changed `intro_price` field in the StoreProduct map to contain the Introductory Price JSON object. `introPrice` has been removed. Removal of `intro_price_cycles`, `intro_price_period`, `intro_price_period_number_of_units`, `intro_price_period_unit`, `intro_price_string` in the iOS map. (https://github.com/RevenueCat/purchases-hybrid-common/pull/147/)
+- Bump RevenueCat dependency (https://github.com/RevenueCat/purchases-hybrid-common/pull/163)
```

Comment:
let's specify whether this is iOS or android, how about "bump `purchases-ios`"?

Reply:
Made it `bump RevenueCat purchases-ios`, you SEO 🙃